### ISSUE TITLE
Can you explain what is meant by triple-linked list?

### ISSUE THREAD

**Issue by maheshsundaram** [WebReflection/linkedom#124]
The README advertised this. I don't see what is meant by it, since in a linear data structure there is only forwards and backwards (single or double linked list). Can you explain what this means?

**Comment by WebReflection** [comment]
there's a *upward* too in here 👋

**Comment by maheshsundaram** [comment]
Upward to where? Is it a different structure than a list then? Because a list has no upward.

**Comment by WebReflection** [comment]
the link moves in 3 directions:
 * forward
 * backward
 * *upward* which is a backward jump, still **the** third link per node

if you see the what, the why, and the how, or the related video/presentation, it explains the meaning of the triple linked list.

is this project about correct computer science terms? **No**

what is the goal of this bug? If it's about correctness of every explanation around the "*third dimension link*" metaphor, please either file a PR with all changes around all descriptions, if you now understand what triple means, or please suggest the purpose of this conversation, considering [I have zero extra time to waste these days](https://twitter.com/WebReflection/status/1490618596271267840), thanks!

**Comment by WebReflection** [comment]
P.S.

  * linked list ... move only forward
  * double linked list ... move forward and backward
  * triple linked list ... fw, bw, and upward

I hope this is not too difficult to grasp but happy to re-iterate if needed.

**Comment by maheshsundaram** [comment]
Hi, hank you for taking your time to explain and congratulations on your child! I wasn't intending any kind of criticism or academic semantics. 

The purpose of this conversation was that I didn't understand what was meant, especially given that (1) it is advertised prominently at the top of the README, (2) is not directly explained in the README, and (3) that you are using this term differently than convention.

Now that I have watched the video from SpeakEasyJS that you mentioned and read the "deep dive", I understand now why you refer to it as a triple-linked list. It's really clever! 

I made a PR for you to consider that may help others like me who had the same question #125
